frontend/wallet/schema.graphql

```diff
@@ -43,50 +13,80 @@ type SyncUpdate {
 
 type SnarkWorker {
   key: String! @examples(values: ["PUB_KEY_E9873DF4453213303DA61F2", "PUB_KEY_F1173DF4453213303DA61F2"])
-  fee: String! @fake(type: money)
+  fee: String! @fake(type: number)
 }
 
-type SnarkFee {
-  snarkCreator: String! @examples(values: ["PUB_KEY_E9873DF4453213303DA61F2", "PUB_KEY_F1173DF4453213303DA61F2"])
-  fee: String! @fake(type: money)
+type UserCommand {
+  id: String!
+  isDelegation: Boolean!
+  nonce: Int!
+  date: String! @fake(type: pastDate)
+  from: String! @examples(values: ["PUB_KEY_E9873DF4453213303DA61F2", "PUB_KEY_F1173DF4453213303DA61F2"]) 
+  to: String! @examples(values: ["PUB_KEY_E9873DF4453213303DA61F2", "PUB_KEY_F1173DF4453213303DA61F2"])
+  amount: String! @fake(type: number)
+  fee: String! @fake(type: number)
+  memo: String @fake(type:hackerPhrase)
+}
+
+type FeeTransfer {
+  recipient: String! @examples(values: ["PUB_KEY_E9873DF4453213303DA61F2", "PUB_KEY_F1173DF4453213303DA61F2"])
```

Comment:
FeeTransfer can come in bundles of one or two. 

Can we create a type called `SingleFeeTransfer`. Then have the schema of FeeTransfer to be:

```graphql
FeeTransfer {
    one : SingleFeeTransfer!
    two: SingleFeeTransfer
}

```

Reply:
Is this correct @deepthiskumar ?

Reply:
Our plan was to split them up here, as the one/two is an implementation detail?

Reply:
> Our plan was to split them up here, as the one/two is an implementation detail?

Oh okay. Then that's fine then.